sharktank/conftest.py

```diff
@@ -81,11 +81,19 @@ def pytest_addoption(parser):
     )
 
     parser.addoption(
-        "--llama3-8b-f16-gguf-path",
+        "--llama3-8b-json-path",
         type=Path,
         action="store",
-        default="/data/extra/models/llama3.1_8B/llama8b_f16.gguf",
-        help="Llama3.1 8b gguf model path, defaults to 30F CI system path",
+        default="/data/extra/models/llama3.1_8B/llama8b_test.json",
```

Comment:
hardcoded path's in a test are not good to have. Can you at least add a todo to resolve in a later pr?